resources/EXM_124/Makefile

```diff
@@ -0,0 +1,49 @@
+all: info .setup-cqf-ruler synthea generate-patients calculate-patients
+
+info:
+	$(info `make` will cache the cqf-ruler, connectathon, and synythea setups, but repeat patient generation and calculation.)
+	$(info To repeat all steps, do `make clean; make`)
+
+.setup-cqf-ruler: .cqf-ruler connectathon .seed-cqf-ruler
+	touch .setup-cqf-ruler
+
+.cqf-ruler:
+	docker run --name cqf-ruler --rm -dit -p 8080:8080 contentgroup/cqf-ruler
+	touch .cqf-ruler
+
+connectathon:
+	git clone --single-branch --branch master https://github.com/DBCG/connectathon.git
+
+.seed-cqf-ruler:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir/Measure \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/resources/fhir3/measure/measure-exm124-FHIR.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/resources/fhir3/library/all-libraries-bundle.json
+	curl -X POST \
+  http://localhost:8080/cqf-ruler-dstu3/fhir \
+  -H 'Content-Type: application/json' \
+	-d @connectathon/resources/fhir3/valuesets/valuesets-bundle-exm124-FHIR.json
```

Comment:
Ugh they reorganized [DBCG/connectathon](https://github.com/DBCG/connectathon), so the paths to the bundles are now broken. The bundles themselves could be changed too, but I haven't checked. Not sure what the best approach is. Should we `git checkout` a specific commit after cloning to ensure we know what we're getting?